src/main/app/src/app/informatie-objecten/informatie-object-edit/informatie-object-edit.component.ts

```diff
@@ -157,13 +157,19 @@ export class InformatieObjectEditComponent implements OnInit, OnDestroy {
         this.subscriptions$.push(verzenddatum.formControl.valueChanges.subscribe(value => {
             if (value && ontvangstDatum.formControl.enabled) {
                 ontvangstDatum.formControl.disable();
-            } else if (!value && ontvangstDatum.formControl.disabled && !this.infoObject.verzenddatum) {
+            } else if (!value && ontvangstDatum.formControl.disabled) {
                 ontvangstDatum.formControl.enable();
             }
         }));
 
-        if (verzenddatum.formControl.value || this.infoObject.verzenddatum) {
-            ontvangstDatum.formControl.disable(verzenddatum.formControl.value);
+        if (ontvangstDatum.formControl.value) {
+            verzenddatum.formControl.disable();
+            //TODO: window.setTimeout verwijderen als angular versie >= 15
```

Comment:
Wel vreemd dat het hier niet werkt maar op regel 172 wel waar eigenlijk hetzelfde gebeurd. 
Als het een bekende bug is in Angular 14 dan graag een Issue aanmaken voor ons zaakafhandelcomponent en daarnaar verwijzen in de todo. En in dat issue graag een link naar de bekende bug van Angular.